src/main/resources/static/css/ui.css

```diff
@@ -255,6 +255,12 @@ body {
 .sidebar .sidebar-nav .nav > .nav-item > .nav-link {
     display: flex;
     font-size: 90%;
+    color: #c1c1c1;
```

Comment:
Looks better with highlighted sections in the nav bar, but for example Documentation Links (or other expandable items) are only highlighted when I expand them. It would be nice to show which is the active element in the navbar by having that link highlighted only. Or the expanded part in the menu oculd be minimized when I click on another element in the navbar.

Reply:
I'm not sure what exactly you are trying to convey. I'm assuming you are don't like the highlighted blocks when sub menus are collapsed/expanded? I tried removing it but thought it looked weird with the minimized bar. I need to understand in greater detail what you want to achieve.

Reply:
I did find another bug though... when you click a sub menu button with a minimzed sidebar, navigo thinks it's an unknown link since there is no data-toggle attached to it to prevent propagation of the event. The link must be disabled completely somehow or navigo will think it is a route.

Reply:
When I select Documentation Links in the left navbar it is expanded and highlighted with a darker color. If I then click on Configure back-end instances directly after, both items are expanded and highlighted. This makes it not clear which one is the latest one I selected. My point is when I clik on a new element which is then expanded the former one should be minimzed again (and not highlighted).

Reply:
Sub menus will now collapse if any other menu item outside the sub menu is clicked.